File: Contribute/Plugin Samples/Plugin Samples/Ruby/XMLParserPlugin/3rd Party Control/IXmlControl.rb
require "mscorlib"
require "System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
require "System.Drawing.Printing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

module XMLTreeView
	# <summary>
	# Summary description for IXmlControl.
	# </summary>
	class IXmlControl
		def Xml
		end

		def Xml=(value)
		end

		def Search(sender, e)
		end

		def StartSearch(criterion, caseSensitive)
		end

		def Next()
		end

		def Print(sender, e)
		end

		def PrintPage(sender, e)
		end

		def PrintDocument
		end

		def Edit(sender, e)
		end

		def Delete(sender, e)
		end

		def Copy(sender, e)
		end

		def Paste(sender, e)
		end

		def Save(sender, e)
		end
	end
end

File: Contribute/Plugin Samples/IronPython/XMLParserPlugin/3rd Party Control/IXmlControl.py
from System import *
from System.Drawing.Printing import *

class IXmlControl(object):
	""" <summary>
	 Summary description for IXmlControl.
	 </summary>
	"""
	def get_Xml(self):

	def set_Xml(self, value):

	Xml = property(fget=get_Xml, fset=set_Xml)

	def Search(self, sender, e):
		pass

	def StartSearch(self, criterion, caseSensitive):
		pass

	def Next(self):
		pass

	def Print(self, sender, e):
		pass

	def PrintPage(self, sender, e):
		pass

	def get_PrintDocument(self):

	PrintDocument = property(fget=get_PrintDocument)

	def Edit(self, sender, e):
		pass

	def Delete(self, sender, e):
		pass

	def Copy(self, sender, e):
		pass

	def Paste(self, sender, e):
		pass

	def Save(self, sender, e):
		pass